src/store/modules/job/actions.ts

```diff
@@ -260,6 +260,96 @@ const actions: ActionTree<JobState, RootState> = {
       showToast(translate("Something went wrong"));
     })
   },
+
+  async fetchDataManagerLogs ({ commit, dispatch }, params) {
+    commit(types.JOB_DATA_MANAGER_LOGS_UPDATED, []);
+    let logs = [] as any
+    const payload = {
+      "inputFields":  {
+        "statusId": ["SERVICE_CANCELLED", "SERVICE_CRASHED", "SERVICE_FAILED", "SERVICE_FINISHED", "SERVICE_PENDING", "SERVICE_RUNNING", "SERVICE_QUEUED"],
+        "statusId_op": "in",
+        "systemJobEnumId_op": "not-empty",
+        "configId": params
+      },
+      "fieldList": ["statusId", "logId", "createdDate", "startDateTime", "finishDateTime", "logFileContentId", "errorRecordContentId", "contentName", "dataResourceId"],
+      "noConditionFind": "Y",
+      "viewSize": 250,
+      "entityName": "DataManagerLogAndContent",
+    }
+
+    await JobService.fetchDataManagerLogs(payload).then((resp: any) => {
+      if (resp.status === 200 && resp.data.docs?.length > 0 && !hasError(resp)) {
+        logs = resp.data.docs
+        dispatch('fetchDataResource', logs)
+      } else {
+        throw resp.data
+      }
+    }).catch((err) => {
+      logger.error(err);
+    })
+    commit(types.JOB_DATA_MANAGER_LOGS_UPDATED, logs);
+    return logs;
+  },
+
+  async fetchDataResource({ commit }, logs) {
+    let dataResourceIds = {};
+    // Extract logFileContentId and errorRecordContentId from logs
+    const contentIds = [].concat(...logs.map((log: any) => [log.logFileContentId, log.errorRecordContentId].filter(id => id)));
+  
+    const payload = {
+      "inputFields": {
+        "coContentId": contentIds
+      },
+      "fieldList": ["coContentId", "coDataResourceId", "coContentName"], 
+      "noConditionFind": "Y",
+      "viewSize": 250,
+      "entityName": "DataResourceContentView"
+    }
+  
+    await JobService.fetchDataResource(payload).then((resp: any) => {
+      if (resp.status === 200 && resp.data.docs?.length > 0 && !hasError(resp)) {
+        dataResourceIds = contentIds.reduce((acc: any, id: string) => {
+          const contentId = resp.data.docs.find((doc: any) => doc.coContentId === id);
+          if (contentId) {
+            acc[id] = {
+              dataResourceId: contentId.coDataResourceId,
+              name: contentId.coContentName
+            };
+          }
+          return acc;
+        }, {});
+      }
+    }).catch((err: any) => {
+      logger.error(err);
+    })
+    commit(types.JOB_DATA_RESOURCE_IDS_UPDATED, dataResourceIds);
```

Comment:
Instead of storing dataResourceIds separately, I think you should add the needed field in the data manager config record itself.